graphql_compiler/schema/schema_info.py

```diff
@@ -388,16 +388,14 @@ class QueryPlanningSchemaInfo:
     # A Statistics object giving statistical information about all objects in the schema.
     statistics: Statistics
 
-    # Dict mapping vertex names in the graphql schema to the Int or ID type property name
-    # to be used for pagination on that vertex. This property should be non-null and
-    # unique for all rows.  An easy choice for pagination key in most situations is the
-    # primary key. The pagination key for a vertex can be omitted making the vertex
-    # ineligible for pagination.
-    #
-    # NOTE(bojanserafimov): The type of this property might be different in the
-    #                       schema graph, due to the process of type overrides that happens
-    #                       during schema generation.
-    pagination_keys: Dict[str, str]
+    # Mapping vertex names in the graphql schema to the non-empty sequence of property names
+    # that are eligible pagination on that vertex in a general context. Some of the provided
+    # properties might be ineligible for pagination in certain queries, for instance if the
+    # query has a "=" filter on the field and it's uniquely indexed. The order of properties
+    # within this list is used as a final (least significant) step in choosing which one
+    # to use when multiple options are available. The pagination key list for a vertex can
+    # be omitted, making the entire vertex ineligible for pagination.
```

Comment:
When you say "omitted" do you mean that the list is empty? If so, let's say that explicitly because it wasn't quite obvious to me.

Reply:
It says above that the sequences are non-empty. I can repeat here if you think it adds value

Reply:
Oh okay so omitted means "not present in this dict"? In that case, how about the phrasing below: